macros/edr/tests/on_run_end/handle_tests_results.sql

```diff
@@ -205,9 +212,15 @@
     {% endif %}
     {% set full_table_name = elementary.insensitive_get_dict_value(recent_anomaly_sample, 'full_table_name', '') %}
     {% set split_full_table_name = full_table_name.split('.') %}
-    {% set database_name = split_full_table_name[0] %}
-    {% set schema_name = split_full_table_name[1] %}
-    {% set table_name = split_full_table_name[2] %}
+    {%- if split_full_table_name | length == 2 %}
```

Comment:
See previous comment.
Anyway making a macro that splits the full_table_name (even if not an adapter macro) is needed (duplicating code) 🙂

Reply:
done